aggregator/options.go

```diff
@@ -47,8 +47,8 @@ var (
 	defaultAggregationSumSuffix      = []byte(".sum")
 	defaultAggregationSumSqSuffix    = []byte(".sum_sq")
 	defaultAggregationMeanSuffix     = []byte(".mean")
-	defaultAggregationLowerSuffix    = []byte(".lower")
-	defaultAggregationUpperSuffix    = []byte(".upper")
+	defaultAggregationMinSuffix      = []byte(".min")
```

Comment:
I'm still not sure about this, if a user uses Min for counters or gauges, he/she will get .min, but if the user uses it for timers, he/she will get ".lower", isn't it confusing?